src/main/java/com/ibm/appscan/bamboo/plugin/impl/SASTScanTask.java

```diff
@@ -90,7 +86,7 @@ private void setUsernameAndPassword(TaskContext taskContext) {
 		String username = credentials.getConfiguration().get("username"); //$NON-NLS-1$
 		scanner.setUsername(username);
 		
-		String password = encryptionService.decrypt(credentials.getConfiguration().get("password")); //$NON-NLS-1$
+		String password = credentials.getConfiguration().get("password"); //$NON-NLS-1$

```

Comment:
@coadaflorin @mattmurp So we're using the password directly here, but I'm concerned whether that is still in encrypted format?  If it's encrypted then then login step would fail.  So obviously if the login works then all is fine (but somewhat surprising bamboo would store passwords in the clear) It's been a while since I worked on this code and I don't have the environment to verify this anymore, could you please confirm whether we indeed get a decrypted password here?

Reply:
It looks like the password it's not encrypted here. I can simply call it and it works. I tried this on my local system and another guy tried it on his system.